tests/test_plotting.py

```diff
@@ -398,45 +459,48 @@ def test_plotly_future_reg():
     future = m.make_future_dataframe(df=df, regressors_df=regressors_df_future, n_historic_predictions=10, periods=50)
     forecast = m.predict(df=future)
 
-    fig1 = m.plot(forecast, plotting_backend="plotly")
-    fig2 = m.plot_components(forecast, plotting_backend="plotly")
-    fig3 = m.plot_parameters(plotting_backend="plotly")
+    fig1 = m.plot(forecast, plotting_backend=plotting_backend)
+    fig2 = m.plot_components(forecast, plotting_backend=plotting_backend)
+    fig3 = m.plot_parameters(plotting_backend=plotting_backend)
 
     if PLOT:
         fig1.show()
         fig2.show()
         fig3.show()
 
 
-def test_plotly_uncertainty():
-    log.info("testing: Plotting with plotly")
+@pytest.mark.parametrize(*decorator_input)
+def test_plot_uncertainty(plotting_backend):
+    log.info(f"testing: Plotting with uncertainty without forecast in focus with {plotting_backend}")
     df = pd.read_csv(PEYTON_FILE, nrows=NROWS)
 
     m = NeuralProphet(epochs=EPOCHS, batch_size=BATCH_SIZE, learning_rate=LR, quantiles=[0.9, 0.1])
```

Comment:
@LeonieFreisinger Can you change the quantiles to `[0.25, 0.75]` for the `test_plotly_uncertainty()`? This is what I originally had in the test_uncertainty.py. You'll also need the change the `m.plot_parameters(quantile=0.75 ...)`, with the exception of the part in `with pytest.raises(ValueError)`. Thx

Reply:
@Kevin-Chen0 thanks a lot for reviewing and for your feedback! 
I adjusted the quantiles accordingly.